scm-ui/ui-components/src/forms/AddKeyValueEntryToTableField.tsx

```diff
@@ -38,12 +39,19 @@ type Props = {
   validateEntry?: (p: string) => boolean;
 };
 
-const FullWidthInputField = styled(InputField)`
-  width: 100%;
+const MobileWrapedDiv = styled.div`
+  @media screen and (min-width: ${devices.tablet.width}px) {
+    & > fieldset {
```

Comment:
This selector is dependent on the internal structure of the InputField component. If we would keep the `FullWidthInputField` as a styled component, we could reference it here so no reference to the `fieldset` would be necessary.